### PR TITLE
tikv: add new config `raftstore.apply-yield-write-size`

### PR DESCRIPTION
**Pull request by glorv**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by glorv** [comment]
waiting for https://github.com/tikv/tikv/pull/13594 to be merged first.

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -812,6 +812,12 @@ raftstore 相关的配置项。
 + 默认值：1024
 + 最小值：大于 0
 
+### `apply-yield-msg-size` <span class="version-mark">从 v6.4.0 版本开始引入</span>
+
++ Apply 线程一轮处理单个状态机的最大字节数，这是一个软限制。
++ 默认值：32KB
+最小值：大于 0
```
Comment (qiancai) [inline tikv-configuration-file.md:819]:
```suggestion
+ 最小值：大于 0
```

File: dynamic-config.md [hunk 2]
```diff
@@ -150,6 +150,7 @@ show warnings;
 | raftstore.merge-check-tick-interval | 触发 Merge 完成检查的时间间隔 |
 | raftstore.cleanup-import-sst-interval | 触发检查过期 SST 文件的时间间隔 |
 | raftstore.local-read-batch-size | 一轮处理读请求的最大个数 |
+| raftstore.apply-yield-msg-size | Apply 线程一轮处理单个状态机写请求的最大数据量 |
```
Comment (BusyJay) [inline dynamic-config.md:153]: 其实这个名字已经不太恰当了。因为现在已经调成了 batch size，也就是名字应该是 apply-yield-write-size，含义是单个状态机在 apply 线程持续写入的最大数据量，这是个软限制。

Comment (glorv) [reply dynamic-config.md]: done

**Review by BusyJay — APPROVED** [review]

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -812,6 +812,12 @@ raftstore 相关的配置项。
 + 默认值：1024
 + 最小值：大于 0
 
+### `apply-yield-write-size` <span class="version-mark">从 v6.4.0 版本开始引入</span>
+
++ 单个状态机在 Apply 线程持续写入的最大数据量，这是个软限制。
++ 默认值：32KB
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:818]:
```suggestion
+ 默认值：32KiB
+ 单位：
```
补充单位的信息

File: dynamic-config.md [hunk 4]
```diff
@@ -150,6 +150,7 @@ show warnings;
 | raftstore.merge-check-tick-interval | 触发 Merge 完成检查的时间间隔 |
 | raftstore.cleanup-import-sst-interval | 触发检查过期 SST 文件的时间间隔 |
 | raftstore.local-read-batch-size | 一轮处理读请求的最大个数 |
+| raftstore.apply-yield-write-size | 单个状态机在 apply 线程持续写入的最大数据量 |
```
Comment (Oreoxmt) [inline dynamic-config.md:153]:
```suggestion
| raftstore.apply-yield-write-size | 单个状态机在 Apply 线程持续写入的最大数据量 |
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -812,6 +812,13 @@ raftstore 相关的配置项。
 + 默认值：1024
 + 最小值：大于 0
 
+### `apply-yield-write-size` <span class="version-mark">从 v6.4.0 版本开始引入</span>
+
++ 单个状态机在 Apply 线程持续写入的最大数据量，这是个软限制。
```
Comment (qiancai) [inline tikv-configuration-file.md:817]:
```suggestion
+ Apply 线程在单个状态机上能持续写入的最大数据量，这是个软限制。
```

Comment (qiancai) [reply tikv-configuration-file.md:817]: 写入的主语是谁呢？英文是 “apply thread can write for one FSM”

Comment (glorv) [reply tikv-configuration-file.md:817]: 👍

File: dynamic-config.md [hunk 6]
```diff
@@ -150,6 +150,7 @@ show warnings;
 | raftstore.merge-check-tick-interval | 触发 Merge 完成检查的时间间隔 |
 | raftstore.cleanup-import-sst-interval | 触发检查过期 SST 文件的时间间隔 |
 | raftstore.local-read-batch-size | 一轮处理读请求的最大个数 |
+| raftstore.apply-yield-write-size | 单个状态机在 Apply 线程持续写入的最大数据量 |
```
Comment (Oreoxmt) [inline dynamic-config.md:153]:
```suggestion
| raftstore.apply-yield-write-size | Apply 线程在单个状态机上能持续写入的最大数据量 |
```

**Pull request merged**